simplelogin/users/myviews.py

```diff
@@ -0,0 +1,38 @@
+from django.contrib import messages
```

Comment:
I think the file name should be only view.py

Reply:
Yes, but views.py was already generated by cookiecutter and in order to separate the logic I made myviews.py and restviews.py files

Reply:
I will merge all in views.py file